src/cli/commands/login/login.ts

```diff
@@ -16,23 +16,42 @@ const defaultScope = `${Environment.AzureCloud.resourceManagerEndpointUrl}/.defa
 
 export async function loginCommand(options: SWACLIConfig) {
   const cmdStartTime = new Date().getTime();
+  const flagsUsed = getFlagsUsed(options);
```

Comment:
instead of calculating flagsUsed everytime can we calcuate it only when we want to log e.g. during logging the telemetry event?

Reply:
No we cant calculate the flagsUsed from the point where we call the sendTelemeteryEvent because we would need the options, available at each of the commands. so calculated at the top and adding to properties when calling the sendtelemetryEvent.

Reply:
ok sounds good